### ISSUE TITLE
Questions about input data normalization

### ISSUE THREAD

**Issue by ymgwjk** [MingboHong/YOLA#7]
Thank you for releasing the code!

I'm a bit confused about how to normalize the input data for a fair comparison with your method. I notice that you normalize the inputs to the range [0, 1] for both YOLOv3 and TOOD, but I can't find pre-trained TOOD models with such an input range in mmdetection. Did you train the baseline model on COCO with other normalization parameters (mean and var) or just load the weights released by mmdetection? And how did you normalize the input data when fine-tuning the TOOD baseline on ExDark?

I would appreciate it if you could help!

**Comment by MingboHong** [comment]
> Thank you for releasing the code!
> 
> I'm a bit confused about how to normalize the input data for a fair comparison with your method. I notice that you normalize the inputs to the range [0, 1] for both YOLOv3 and TOOD, but I can't find pre-trained TOOD models with such an input range in mmdetection. Did you train the baseline model on COCO with other normalization parameters (mean and var) or just load the weights released by mmdetection? And how did you normalize the input data when fine-tuning the TOOD baseline on ExDark?
> 
> I would appreciate it if you could help!

Hi， I trained both detectors with a range [0, 1] (YOLOv3 TOOD) from scratch on MSCOCO (as shown in Table 4). You can find the pre-trained ckpt in my Google Drive.

However, when I conduct the fine-tuning task on the downstream dataset (e.g., exdark darkface), I just employ the weights released by mmdetection for a fair comparison. But I changed the config as follows:
'''
data_preprocessor = dict(
    type='DetDataPreprocessor',
    mean=[0, 0, 0],
    std=[255., 255., 255.],
    bgr_to_rgb=True,
    pad_size_divisor=32)
'''
However, you also can consider using the YOLA detector pre-trained on the MSCOCO dataset as a base model (range [0, 1]), and fine-tune it on the downstream dataset.

We observe that the TOOD detector will be further improved (DarkFace 67.5 V.S. 68.9; ExDark 75.3 V.S. 75.5)
